src/form.js

```diff
@@ -0,0 +1,131 @@
+import { addTasktoTasks} from "./tasks"
+import { taskLog } from "./taskLog"
+
+const newTodo = () =>{
```

Comment:
This function body is too long. Please consider splitting it.
Can have separate functions that return the elements:
`const titleLabel = getLabel()
form.appendChild(titleLabel)`